compiler-rt/lib/scudo/standalone/primary64.h

```diff
@@ -530,7 +530,7 @@ template <typename Config> class SizeClassAllocator64 {
 
   struct ReleaseToOsInfo {
     uptr BytesInFreeListAtLastCheckpoint;
-    uptr RangesReleased;
+    uptr NumReleaseAttempted;
```

Comment:
Can you change the name to NumReleasesAttempted here too?

Reply:
Done.